apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2237,19 +2284,19 @@ static void tc_libc_stdio_memsostream(void)
 
 	offset = memsostream.public.seek((FAR struct lib_sostream_s *)&memsostream.public, SEEK_OFFSET, SEEK_SET);	/* Seek from the start of the file */
 	TC_ASSERT_EQ("memsostream_seek", memsostream.offset, SEEK_OFFSET);
-	TC_ASSERT_EQ("memsostream_seek", offset, (off_t)SEEK_OFFSET);
+	TC_ASSERT_EQ("memsostream_seek", offset, (off_t) SEEK_OFFSET);
```

Comment:
revert. Please check all of this kind of change.

Reply:
Reverted all. 
Sorry. This change was done by the formatter tool.